hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+In preparation for supporting JAX in :ref:`hypothesis.extra.array_api <array-api>`,
```

Comment:
For future reference, we have a neat custom role `:pypi:'jax'` which will... link to the package's PyPI page.  No need to add that here though.